{"name": "test_corteva_title_api", "status": "broken", "statusDetails": {"message": "pyppeteer.errors.BrowserError: Browser closed unexpectedly:", "trace": "@pytest.fixture(scope=\"module\")\n    def get_job_details():\n        \"\"\"\n        Fixture for scraping process from the career section.\n        \"\"\"\n>       scraper_data = cortevaScraper()\n\ntests/Test_API_Suite/test_corteva.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nscrapers/corteva_scraper.py:27: in scraper\n    soup = GetDynamicSoup(\"https://careers.corteva.com/job-search-results/?keyword=Romania\")\nscrapers/__utils/dynamic_requests_html_shorts.py:33: in __new__\n    response.html.render()\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests_html.py:586: in render\n    self.browser = self.session.browser  # Automatically create a event loop and browser\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests_html.py:730: in browser\n    self._browser = self.loop.run_until_complete(super().browser)\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/asyncio/base_events.py:649: in run_until_complete\n    return future.result()\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/requests_html.py:714: in browser\n    self._browser = await pyppeteer.launch(ignoreHTTPSErrors=not(self.verify), headless=True, args=self.__browser_args)\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/pyppeteer/launcher.py:307: in launch\n    return await Launcher(options, **kwargs).launch()\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/pyppeteer/launcher.py:168: in launch\n    self.browserWSEndpoint = get_ws_endpoint(self.url)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nurl = 'http://127.0.0.1:52031/json/version'\n\n    def get_ws_endpoint(url) -> str:\n        url = url + '/json/version'\n        timeout = time.time() + 30\n        while (True):\n            if time.time() > timeout:\n>               raise BrowserError('Browser closed unexpectedly:\\n')\nE               pyppeteer.errors.BrowserError: Browser closed unexpectedly:\n\n/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/pyppeteer/launcher.py:227: BrowserError"}, "start": 1706505222123, "stop": 1706505222123, "uuid": "4788125d-0040-42d8-97ae-063971d05cf6", "historyId": "49d82d4d6ba52dcd8290ba2d3957c6c9", "testCaseId": "49d82d4d6ba52dcd8290ba2d3957c6c9", "fullName": "Test_API_Suite.test_corteva#test_corteva_title_api", "labels": [{"name": "tag", "value": "API"}, {"name": "tag", "value": "regression"}, {"name": "parentSuite", "value": "Test_API_Suite"}, {"name": "suite", "value": "test_corteva"}, {"name": "host", "value": "fv-az1109-305"}, {"name": "thread", "value": "1947-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_API_Suite.test_corteva"}]}